UndertaleModTool/Scripts/Resource Unpackers/ExportSpecificSprites.csx

```diff
@@ -0,0 +1,67 @@
+//Basically a mashup of ExportAllSprites and DumpSpecificCode
+using System.Text;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UndertaleModLib.Util;
+
+EnsureDataLoaded();
+
+bool padded = (ScriptQuestion("Export all sprites WITH padding?"));
+
+string texFolder = PromptChooseDirectory();
+if (texFolder == null)
+    throw new ScriptException("The export folder was not set.");
+texFolder = texFolder + "Sprites" + Path.DirectorySeparatorChar;

```

Comment:
This was a copy of how it was done on ExportAllSprites.csx

Reply:
yeah but just because some other scripts have bad coding habits, doesn't mean that they should be copied over :sweat_smile:

Reply:
Using PathCombine leads to the sprites ending up on the chosen folder instead of the chosenfolder/Sprites

Reply:
Using the texFolder + "Sprites" + Path.DirectorySeparatorChar way ends up on the intended behaviour, the sprites end up on the chosenfolder/Sprites

Reply:
> Using the texFolder + "Sprites" + Path.DirectorySeparatorChar way ends up on the intended behaviour, the sprites end up on the chosenfolder/Sprites

The reason why it ended up on teh chosefolder before, is because you had a typo. you had `codeFolder = ...` instead of `texFolder = ...`
![grafik](https://github.com/krzys-h/UndertaleModTool/assets/38186597/1d278c6c-552b-404c-abc5-eacd4cba90fd)

Reply:
I know the second commit had that typo, but leaving it like that just made the script fail so I realized it an I corrected it
![image](https://github.com/krzys-h/UndertaleModTool/assets/71221709/cf4ae932-0c8b-4155-bc31-768f52446c52)
Once corrected it has that error of not exporting in the correct folder

Reply:
I realized the reason this happens is because when using path.combine it doesn't add a at the end, while the old code did add it
(Screenshot of the final message with the old code)
![image](https://github.com/krzys-h/UndertaleModTool/assets/71221709/897987fe-4bb9-4be3-9831-49a4ea1a40aa)

(Screenshot of the final message with the new code)
![image](https://github.com/krzys-h/UndertaleModTool/assets/71221709/965240b5-a1c3-4059-858a-0e6c96e4009c)

Reply:
> I realized the reason this happens is because when using path.combine it doesn't add a \ at the end, while the old code did add it

This is normal and usually is fine. What you're also missing, is a path.combine in line 66 (`worker.ExportAsPNG(sprite.Textures[i].Texture, texFolder + sprite.Name.Content + "_" + i + ".png", null, padded);`)

Reply:
Oh I see, makes sense now, thanks :3